spark/src/main/scala/geotrellis/spark/reproject/TileRDDReproject.scala

```diff
@@ -60,7 +60,8 @@ object TileRDDReproject {
     metadata: TileLayerMetadata[K],
     destCrs: CRS,
     targetLayout: Either[LayoutScheme, LayoutDefinition],
-    options: Options
+    options: Options,
+    sc: Option[SparkContext] = None
```

Comment:
Can't add parameter in order to keep the API. But you can grab `SparkContext` from `bufferedTiles.context`.